Header header

float32[3] ranges
float32[3] accel
float32[3] global_accel
float32[3] global_accel_avg
float32[3] global_vel_avg
